### DISCUSSION TITLE
rust-course/4.1章节，编写测试及控制执行，最后有个链接失效了

### DISCUSSION THREAD

**Discussion by digger-yu** [sunface/rust-course#520] [内容建议]
![图片](https://user-images.githubusercontent.com/55081697/156693270-8da7444e-95fe-4ce3-99d3-51a83718772d.png)
![图片](https://user-images.githubusercontent.com/55081697/156693306-a335e9c4-7839-4352-abc2-2010ef48259a.png)

**Comment by sunface** [comment]
好，我看看，感谢提醒。

**Comment by sunface** [comment]
已解决。。
